esrally/mechanic/supplier.py

```diff
@@ -865,8 +865,16 @@ def tail_container_logs(self, output, container_name):
 
     def check_container_return_code(self, completion, container_name):
         if completion["StatusCode"] != 0:
-            raise SystemSetupError(
-                f"Docker container [{container_name}] failed with status code [{completion['StatusCode']}]: Error [{completion['Error']}]"
+            msg = "Executing '{}' failed. The last 20 lines in the build log file are:\n".format(container_name)
+            msg += "=========================================================================================================\n"
+            with open(self.log_file, "r", encoding="utf-8") as f:
+                msg += "\t"
+                msg += "\t".join(f.readlines()[-20:])
+            msg += "=========================================================================================================\n"
+            msg += "The full build log is available at [{}]".format(self.log_file)
```

Comment:
same comment, let's stick to f-strings

Reply:
Addressed in https://github.com/elastic/rally/pull/1629/commits/efd06ea3e9ba685e87015815d7e8827176bb0b28